app/views/providers/open_banking_guidances/show.html.erb

```diff
@@ -12,17 +12,10 @@
     <p class="govuk-body"><%= t(".your_client_can") %></p>
     <p class="govuk-body"><%= t(".its_a_fast") %></p>
 
-    <details class="govuk-details govuk-!-display-none-print" data-module="govuk-details">
-      <summary class="govuk-details__summary">
-          <span class="govuk-details__summary-text">
-            <%= t(".details_summary") %>
-          </span>
-      </summary>
-      <div class="govuk-details__text govuk-!-padding-bottom-1">
-        <%= t(".details_content_html") %>
-        <%= list_from_translation_path(".open_banking_consents.show.details") %>
-      </div>
-    </details>
+    <%= govuk_details(summary_text: t(".details_summary")) do %>
```

Comment:
Layout: is the layout ok without the override class?

Reply:
I have added this back in! Thank you